hypervisor/src/hypervisor/config.rs

```diff
@@ -38,6 +39,20 @@ pub struct Partition {
     pub devices: Vec<Device>,
     #[serde(default)]
     pub hm_table: PartitionHMTable,
+    #[serde(default)]
+    pub udp_ports: Vec<SockAddr>,
+}
+
+/// A port the partition should listen on.
```

Comment:
This comment looks a bit misplaced? Why store port in a string? I think either the comment or the code is wrong in that place.